tests/test_fft.py

```diff
@@ -3,6 +3,12 @@
 
 import dpnp
 
+from .helper import has_support_aspect64
```

Comment:
looks as never used

Reply:
This is used on the next line
`
pytestmark = pytest.mark.skipif(
    not has_support_aspect64(), reason="Aborted on Iris Xe"
)
`

Reply:
but where is `pytestmark` used?

Reply:
to skip all tests in a module based on some condition

Reply:
then you need to add a reference on opened issue to keep track of the problem

Reply:
done